BaseLib/FileFinder.h

```diff
@@ -36,7 +36,8 @@ class FileFinder
     FileFinder()
     {
         addDirectory(".");
-        addDirectory(BuildInfo::source_path + "/FileIO");
+        addDirectory(BuildInfo::source_path + "/GeoLib/IO/XmlIO");
```

Comment:
i don't agree with you. Configuration files are application specific. If FileFinder is only used in that way, I would move this to ApplicationsLib and rename it to `ConfigFieFinder`.